docker-compose.yml

```diff
@@ -18,7 +17,7 @@ services:
       - db
     # Mount the app folder to increase development speed. Ignore compiled and build folders.
     volumes:
-      - .:/app
+      # - .:/app
```

Comment:
add a comment here why this is uncomment, e.g.: nice for dev purposes for fast exchange of files, but does not work with prod

Reply:
Removed it because it was a mistake